opm/simulators/wells/SingleWellState.hpp

```diff
@@ -55,6 +67,16 @@ class SingleWellState {
     Well::ProducerCMode production_cmode{Well::ProducerCMode::CMODE_UNDEFINED};
 
 
+    /// Special purpose method to support dynamically rescaling a well's
+    /// CTFs through WELPI.
+    ///
+    /// \param[in] well_perf_data New perforation data.  Only
```

Comment:
Tiny nit: The documented parameter name should match the actual parameter name.

Reply:
OK